src/libraries/System.IO.Pipes/src/System.IO.Pipes.csproj

```diff
@@ -3,11 +3,14 @@
     <AssemblyName>System.IO.Pipes</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <OmitTransitiveCompileReferences>true</OmitTransitiveCompileReferences>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
Why compile it with `AnyOS` instead? I would like to be intentional of shipping not supported on the platforms it is not supported.

Reply:
I think this was the convention we landed on in https://github.com/dotnet/runtime/pull/37944, so any future platform that's not Unix/Windows would also get the PNSE behavior by default.

/cc @Anipik

Reply:
Yeah this should be fine. I just looked at: https://github.com/dotnet/runtime/blob/master/eng/targetframeworksuffix.props 

and the only platform that will produce a PNSE is `Browser` so either way is fine.